src/Microsoft.DotNet.Darc/src/Darc/Options/TriggerSubscriptionsCommandLineOptions.cs

```diff
@@ -13,6 +15,9 @@ internal class TriggerSubscriptionsCommandLineOptions : SubscriptionsCommandLine
         [Option("id", HelpText = "Trigger a specific subscription by id.")]
         public string Id { get; set; }
 
+        [Option("build", HelpText = "If specified, selects a specific BAR build id to use; otherwise will use the latest available from the supplied '--source-repo'")]
```

Comment:
Once this is merged, could you open a follow up pr to update https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md#trigger-subscriptions with the new param?

Reply:
will do!

Reply:
https://github.com/dotnet/arcade/pull/6081